src/unit_tests/os_crypto/shared/test_msgs.c

```diff
@@ -269,6 +264,65 @@ void test_StoreCounter_fail_first_open(void **state)
     os_free(keys.keyentries);
 }
 
+void test_encrypt_by_method_blowfish(void **state){
+    const char *key = "test_key";
+    const char *string = "test string";
+    const int buffersize = 1024;
+    char buffer1[buffersize];
+    char buffer2[buffersize];
+
+    doEncryptByMethod(string, buffer1, key, strlen(string), OS_ENCRYPT ,W_METH_BLOWFISH);
```

Comment:
The return value of function "doEncryptByMethod" could also be asserted in the test

Reply:
done